docs/api-constructor.md

```diff
@@ -20,7 +20,7 @@ Implements the [stream.Duplex][1] class.
     JPEG, PNG, WebP, AVIF, GIF, SVG, TIFF or raw pixel image data can be streamed into the object when not present.
 *   `options` **[Object][13]?** if present, is an Object with optional attributes.
 
-    *   `options.failOn` **[string][12]** level of sensitivity to invalid images, one of (in order of sensitivity): 'none' (least), 'truncated', 'error' or 'warning' (most), highers level imply lower levels. (optional, default `'warning'`)
+    *   `options.failOn` **[string][12]** Level of sensitivity to invalid images in terms of pixel values. Invalid image headers/metadata will always fail regardless of this setting. One of (in order of sensitivity): 'none' (least), 'truncated', 'error' or 'warning' (most), highers level imply lower levels. (optional, default `'warning'`)
```

Comment:
I don't understand enough of sharp to better word this sentence:

> Level of sensitivity to invalid images in terms of pixel values.

This needs more clarity